docs/operations/memory_dump.md

```diff
@@ -13,9 +13,16 @@ The memory dump process mounts a PVC to the virt-launcher in order to get the ou
 [feature gates](./activating_feature_gates.md#how-to-activate-a-feature-gate)
 field in the KubeVirt CR must be expanded by adding the `HotplugVolumes` to it.
 
-### A PVC to hold the memory dump output
+## Virtctl support
+
+### Get memory dump
+
+Now lets assume we have a running VM and the name of the VM is 'my-vm'.
+We can either dump to an existing pvc, or request one to be created.
```

Comment:
I think it looks and reads better with "You" instead of "We" in documentation.

But I am not a native speaker, so do take my advice with a grain of salt.

Reply:
yeah I thought about it, the 'lets assume we have' I took from another documentation and (also in this PR I just moved it, it is already in the original doc). Since the 'lets assume it made me write the entire doc in 'we' instead of 'you'.
I'll try to rephrase..